pkg/scheduler/partition_test.go

```diff
@@ -123,7 +123,6 @@ func TestNewWithPlacement(t *testing.T) {
 			},
```

Comment:
I'll look into this, but the further up the chain we go, the more difficult the setup gets as more of the environment needs to be mocked. This gets even more difficult in the case of preemption due to the predicate checks. It may not be a worthwhile effort.